docs/source/markdown/options/network.md

```diff
@@ -62,11 +60,20 @@ Valid _mode_ values are:
     container to host using the gateway address. The latter can be overridden
     by passing **--map-gw** in the pasta-specific options (despite not being an
     actual pasta(1) option). \
+    For better integration with DNS handling **--dns-forward 169.254.1.1** is passed
+    and the ip is added to resolv.conf as first entry. It is possible to pass **--dns-forward**
+    explicitly in case a different ip should be used.
+    To make the `host.containers.internal` /etc/hosts entry work and allow connections
+    to the host **--map-guest-addr 169.254.1.2** is passed as well, again it can be set
```

Comment:
```suggestion
    to the host, **--map-guest-addr 169.254.1.2** can also be passed. Again, it can be set
```

Reply:
It is not "can", it is set by default. It can also be passed by a user to overwrite it but podman will always set this as default option. I changed to just "is passed" like in the first sentence.